### DISCUSSION TITLE
OrchardCore Deployment

### DISCUSSION THREAD

**Discussion by sea-4** [OrchardCMS/OrchardCore#9553] [Q&A]
What is the best way to deploy a site from local development to remote live server initially? Should I use Web Deploy and export/import database using sql  script?

**Comment by Piedone** [comment]
You'll need to deploy the app in any case the first time, so a Web Deploy deployment or something else is needed. Then, you'll need to transfer the content you've created locally to that site: The cleanest I think is to use recipes indeed.
